client/src/components/DueDate/DueDate.test.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import DueDate from './DueDate';
+
+const months = [
+  'JAN',
+  'FEB',
+  'MAR',
+  'APR',
+  'MAY',
+  'JUN',
+  'JUL',
+  'AUG',
+  'SEP',
+  'OCT',
+  'NOV',
+  'DEC',
+];
+
+describe('<DueDate />', () => {
+  it('should render correct date', () => {
+    const dateString = '2019-06-07T23:45:33.293+00:00';
+    const date = new Date(dateString).getDate().toString();
```

Comment:
instead of evaluation the date here, just hard code it, so that you know that what will be the expected output

Reply:
done